src/docs/docs_summarizer.py

```diff
@@ -1,11 +1,13 @@
 import os
+
 import llama_index
```

Comment:
why is this still here (l) if we are alpha sorting, and then you left the other llama-related a few lines down?

Reply:
please look below:
Ideal behavior is:

    "future" imports first
    sort default packages
    sort third party packages
    sort in-project packages

Reply:
:+1: